skafka/src/main/scala/com/evolutiongaming/skafka/producer/ProducerConfig.scala

```diff
@@ -29,6 +29,7 @@ final case class ProducerConfig(
   transactionTimeout: FiniteDuration                = 1.minute,
   transactionalId: Option[String]                   = None,
   saslSupport: SaslSupportConfig                    = SaslSupportConfig.Default,
+  sslSupport: SslSupportConfig                      = SslSupportConfig.Default,
```

Comment:
this is not backward compatible change, could you please use same approach as here?
https://github.com/evolution-gaming/scassandra/blob/master/scassandra/src/main/scala/com/evolutiongaming/scassandra/CassandraConfig.scala#L37